VogtObserver4/Program.cs

```diff
@@ -1,12 +1,34 @@
 ﻿using System;
-
+using System.Threading;
 namespace VogtObserver4
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Install a sensor at Reactor A
+            RadiationSensor sensor = new RadiationSensor("Reactor A", 10);
```

Comment:
Usually you would parameterize a seed. 

Using the same exact seed should produce the same exact results with the random number generator.

Reply:
Not a big deal but something to be aware of if you dont change it